incubator/piwik/templates/apache-pvc.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.persistence.enabled -}}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  name: {{ template "fullname" . }}-apache
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.persistence.apache.storageClass | quote }}
```

Comment:
We now add alpha and beta storageclass annotation support, see
https://github.com/kubernetes/charts/blob/master/stable/osclass/templates/apache-pvc.yaml#L11-L16. Similar change is required in the piwiki pvc volume spec as well